entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+cd web
+if [ "$RUN_MIGRATION" -eq "1" ]; then
+	echo "RUN MIGRATION is TRUE"
```

Comment:
[nitpick]
You *might* want to log this to stderr.

```suggestion
	>&2 echo "RUN MIGRATION is TRUE"
```